source/designers-guide/configuration-using-theme-php/index.md

```diff
@@ -251,6 +251,31 @@ Now you can either override the default unique names like e.g. `brand-primary` o
 Every existing field which is not overwritten in your <code>Theme.php</code> will use the default value of your parent theme.
 </div>
 
+## Create a theme info tab
+
+In order to create a new tab containing information about your theme, you create a new directory in your theme folder called **info**. In that directory you can add HTML files named like the language ISO you support.
+In that file u can write and style 
+
+![Configuration set](info-directory.jpg)
+
+If the current backend language ISO is not found, Shopware uses the fallback en_GB.html. 
+
+In this file, you can use HTML and CSS to make your information as you want.
```

Comment:
Use HTML and CSS in your file to style and describe the information as you want.